### PR TITLE
Update Asset Store and HCMS docs in the community repo

### PR DESCRIPTION
**Pull request by mmitoraj**
Changes proposed in this pull request:

- Consolidate Asset Store and HCMS docs in the community repo

**Related issue(s)**
See also /kyma-project/rafter/issues/10

### CODE REVIEW

File: guidelines/content-guidelines/09-add-documentation-topic.md [hunk 1]
```diff
@@ -3,86 +3,85 @@ title: Add new documentation to the website
...
+
 ## Add a new documentation topic
-​
+
 Follow these steps:
 
 1. Create a pull request with `.md` files for the new documentation topic. Place the `.md` files under a new `docs` subfolder in the `kyma` repository, such as `docs/api-gateway-v2/`.
 
-2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterDocsTopic CR](https://kyma-project.io/docs/components/headless-cms/#custom-resource-cluster-docs-topic) for your topic. For example, if you add a ClusterDocsTopic CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cdt.yaml`. ​
+2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterAssetGroup CR](https://kyma-project.io/docs/components/content-management/#custom-resource-cluster-asset-group) for your topic. For example, if you add a ClusterAssetGroup CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cag.yaml`. ​
```
Comment (kazydek) [inline guidelines/content-guidelines/09-add-documentation-topic.md]:
```suggestion
2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterAssetGroup CR](https://kyma-project.io/docs/components/rafter/#custom-resource-cluster-asset-group) for your topic. For example, if you add a ClusterAssetGroup CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cag.yaml`. ​
```

File: guidelines/releases-guidelines/02-kyma-release-process.md [hunk 2]
```diff
@@ -89,20 +89,20 @@ Follow these steps to release another Kyma version.
    FROM {kyma_operator_path}/kyma-operator:{kyma_operator_version}
    ```
 
-   iii. In the `resources/core/values.yaml` file, replace the the `clusterDocsTopicsVersion` value with your release branch name. For example, for the 1.4 release, find the following section:
+   iii. In the `resources/core/values.yaml` file, replace the the `clusterAssetGroupsVersion` value with your release branch name. For example, for the 1.4 release, find the following section:
```
Comment (kazydek) [inline guidelines/releases-guidelines/02-kyma-release-process.md]:
```suggestion
   iii. In the `resources/core/values.yaml` file, replace the `clusterAssetGroupsVersion` value with your release branch name. For example, for the 1.4 release, find the following section:
```

File: guidelines/templates/resources/metrics.md [hunk 3]
```diff
@@ -44,4 +44,4 @@ curl http://localhost:{port}/{endpoint}
 
 See the [Monitoring](/components/monitoring) documentation to learn more about monitoring and metrics in Kyma.
 
-> For reference, see the existing **Metrics** documents for the [Asset Store](https://kyma-project.io/docs/1.6/components/asset-store/#metrics-metrics).
+> For reference, see the existing **Metrics** documents for the [Rafter]](https://kyma-project.io/docs/1.9/components/rafter/#metrics-metrics).
```
Comment (kazydek) [inline guidelines/templates/resources/metrics.md]:
```suggestion
> For reference, see the existing **Metrics** documents for the [Rafter](https://kyma-project.io/docs/1.9/components/rafter/#metrics-metrics).
```

File: guidelines/content-guidelines/09-add-documentation-topic.md [hunk 4]
```diff
@@ -3,86 +3,85 @@ title: Add new documentation to the website
...
+
 ## Add a new documentation topic
-​
+
 Follow these steps:
 
 1. Create a pull request with `.md` files for the new documentation topic. Place the `.md` files under a new `docs` subfolder in the `kyma` repository, such as `docs/api-gateway-v2/`.
 
-2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterDocsTopic CR](https://kyma-project.io/docs/components/headless-cms/#custom-resource-cluster-docs-topic) for your topic. For example, if you add a ClusterDocsTopic CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cdt.yaml`. ​
+2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterAssetGroup CR](https://kyma-project.io/docs/components/rafter/#custom-resource-cluster-asset-group) for your topic. For example, if you add a ClusterAssetGroup CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cag.yaml`. ​
```
Comment (magicmatatjahu) [inline guidelines/content-guidelines/09-add-documentation-topic.md]: Path will change:

```md
2. In the same PR, create a `.yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/content-ui/templates) folder to add a [ClusterAssetGroup CR](https://kyma-project.io/docs/components/rafter/#custom-resource-cluster-asset-group) for your topic. For example, if you add a ClusterAssetGroup CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cag.yaml`. ​
```

File: guidelines/content-guidelines/09-add-documentation-topic.md [hunk 5]
```diff
@@ -3,86 +3,85 @@ title: Add new documentation to the website
...
-​
+
 In addition to Kyma documentation, there are also [OpenAPI](https://swagger.io/specification/) specifications rendered on the [Kyma website](https://kyma-project.io). You can find these specifications under the **API Consoles** type in the right navigation panel of a given component topic.
 ​
-
 To add a new specification, follow these steps:
-​
 
-1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder and locate an existing ClusterDocsTopic CR that you want to modify.
+1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder and locate an existing ClusterAssetGroup CR that you want to modify.
```
Comment (magicmatatjahu) [inline guidelines/content-guidelines/09-add-documentation-topic.md]: Path will change:

```md
1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/content-ui/templates) folder and locate an existing ClusterAssetGroup CR that you want to modify.
```

File: guidelines/content-guidelines/09-add-documentation-topic.md (hunk 4 above)
Comment (magicmatatjahu) [reply guidelines/content-guidelines/09-add-documentation-topic.md]: Link `https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates` is correct. Sorry.

File: guidelines/content-guidelines/09-add-documentation-topic.md (hunk 5 above)
Comment (magicmatatjahu) [reply guidelines/content-guidelines/09-add-documentation-topic.md]: Link `https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates` is correct. Sorry.

**Pull request merged**